lib/matplotlib/tests/test_quiver.py

```diff
@@ -74,6 +74,20 @@ def test_quiver_with_key():
                                       'size': 'large'})
 
 
+@image_comparison(baseline_images=['quiver_orientation_test_image'],
```

Comment:
Best remove any text from this test. It will make it more resilient to freetype changes.